modules/identify-dockerfile-line-component-cve.adoc

```diff
@@ -13,12 +13,12 @@ You can identify specific Dockerfile lines in an image that introduced component
 
 To view a problematic line:
 
-. Go to the {product-title-short} portal and click *Vulnerability Management* from the navigation menu.
+. Go to the {product-title-short} portal and click *Vulnerability Management* -> *Dashboard* from the navigation menu.
 . Select an image from either the *Top Riskiest Images* widget or click the *Images* button at the top of the Dashboard and select an image.
 . In the *Image* details view, under *Image Findings*, CVEs are listed in the *Observed CVEs*, *Deferred CVEs*, and *False positive CVEs* tabs.
```

Comment:
This split is no longer available under Dashboard and the user should go to Workload CVEs to list deferred and False positive CVEs. Also, the user must use the Workload CVEs view to submit an exception request to defer new CVEs or mark them False positive. After that the requests can be approved / denied by a user with appropriate permissions from Vulnerability Management -> Exception Management view.